packages/augur-ui/src/modules/reporting/common.styles.less

```diff
@@ -589,6 +589,10 @@ div.ReportingSubheaders {
     .text-16-medium;
 
     color: @color-primary-text;
+
+    > a {
+      word-break: break-word;
```

Comment:
Fixes reporting cards for lower resolutions (smartphones).

They were overflowing to the right, making the report button unclickable.

Reply:
Lgtm